deployments/liqo/README.md

```diff
@@ -34,8 +34,7 @@
 | crdReplicator.imageName | string | `"liqo/crd-replicator"` | crdReplicator image repository |
 | crdReplicator.pod.annotations | object | `{}` | crdReplicator pod annotations |
 | crdReplicator.pod.labels | object | `{}` | crdReplicator pod labels |
-| discovery.config.autojoin | bool | `true` | Automatically join discovered cluster exposing the Authentication Service with a valid certificate |
-| discovery.config.autojoinUntrusted | bool | `true` | Automatically join discovered cluster exposing the Authentication Service with a self-signed certificate |
+| discovery.config.autojoin | bool | `true` | Automatically join discovered clusters |
```

Comment:
@palexster Is "join" different from "peer"? In case they are synonyms, I would suggest to use the same term all across the project to avoid confusion.